includes/class-wp-job-manager-ajax.php

```diff
@@ -139,7 +139,7 @@ public function get_listings() {
 			'orderby'            => $orderby,
 			'order'              => sanitize_text_field( $_REQUEST['order'] ),
 			'offset'             => ( absint( $_REQUEST['page'] ) - 1 ) * absint( $_REQUEST['per_page'] ),
-			'posts_per_page'     => absint( $_REQUEST['per_page'] ),
+			'posts_per_page'     => max( 1, absint( $_REQUEST['per_page'] ) ),
```

Comment:
As a behavior change, I might cherry-pick this commit to a separate PR, if that works for you? There could be some themes doing things they shouldn't with `-1`.

Reply:
Sounds good

Reply:
I started doing this and then realized `absint()` prevented `-1` 😉 . Seems redundant.

Reply:
eh but it does prevent passing `0`, I wanted to add a `min` too so there was an upper limit but i figured that would be problematic

Reply:
In testing, `0` resulted in the default `per_page`, but oh well.